src/libcollections/fmt.rs

```diff
@@ -505,12 +505,24 @@ use string;
 ///
 /// # Examples
 ///
+/// Basic usage:
+///
 /// ```
 /// use std::fmt;
 ///
 /// let s = fmt::format(format_args!("Hello, {}!", "world"));
 /// assert_eq!(s, "Hello, world!".to_string());
 /// ```
+///
+/// Please note that using [format!][format_macro] might be preferrable.
```

Comment:
format! should be wrapped in backticks so that it can look like code. Also, the `[format_macro]` part shouldn't be needed, `[format!]` should work just fine.